File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        res = []

        l = 0
        r = k - 1
        
        while r < len(nums):
            subList = nums[l:r+1]
            res.append(max(subList))

            l += 1
            r += 1

        return res

File: Data Structures & Algorithms/first-missing-positive/submission-0.cpp
class Solution {
public:
    int firstMissingPositive(vector<int>& nums) {
        int missing = 1;
        while (true) {
            bool flag = true;
            for (int i = 0; i <nums.size(); i++) {
                if (nums[i] == missing) {
                    flag = false;
                    break;
                }
            }

            if (flag == true) {
                return missing;
            }
            missing++;
        }
    }
};